packages/wonder-blocks-grid/components/cell.js

```diff
@@ -39,6 +39,13 @@ type Props = {|
     style?: StyleType,
 |};
 
+type DefaultProps = {|
+    smallCols: $PropertyType<Props, "smallCols">,
+    mediumCols: $PropertyType<Props, "mediumCols">,
+    largeCols: $PropertyType<Props, "largeCols">,
+    cols: $PropertyType<Props, "cols">,
```

Comment:
These props in `Props` are optional, but we're providing default values for them so they should be required.

Reply:
Possibly, but I don't think it matters in a types first world.